programmer-humour/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adrian Ilovan</title>
+    <style>
+        #image {
+            max-width: 100%;
+            height: auto;
+        }
+    </style>
```

Comment:
You should try seperating this into a style.css file. and then linkng it using
```html
<link rel="stylesheet" href="style.css" />
```

Reply:
Hi, thanks for reviewing my code :)
I misunderstood the requirements and thought we are supposed to do them in only one file.